runtime/lua/vim/lsp.lua

```diff
@@ -1587,6 +1587,13 @@ function lsp.formatexpr(opts)
   return 0
 end
 
+-- Provides an interface between the built-in client and `tagfunc`
+--
+-- Used via `set tagfunc=v:lua.vim.lsp.tagfunc`
+function lsp.tagfunc(...)
+  return require('vim.lsp.tagfunc')(...)
```

Comment:
In this case the `tagfunc.lua` module should just return the `tagfunc` function, rather than a table containing the function. I think this is fine since we aren't providing any options or anything in the `tagfunc.lua` module so returning the function directly makes sense.